ui/selectors/transactions.js

```diff
@@ -596,3 +600,21 @@ export function hasTransactionPendingApprovals(state) {
     TRANSACTION_APPROVAL_TYPES.some((type) => hasPendingApprovals(state, type))
   );
 }
+
+const ConfirmationApprovalTypes = [
+  ApprovalType.EthSign,
+  ApprovalType.PersonalSign,
+  ApprovalType.EthSignTypedData,
+  ApprovalType.Transaction,
+];
```

Comment:
Q: can you help me understand the differences between ConfirmationApprovalTypes and TRANSACTION_APPROVAL_TYPES?

Reply:
The set of ApprovalTypes are different between the 2.

Reply:
curious, why?